### ISSUE TITLE
Will you support fine-tuning from olmo2?

### ISSUE THREAD

**Issue by zhudefa** [allenai/open-instruct#467]
Great job on this impressive work!

Will you support fine-tuning from olmo2?

**Comment by vwxyzjn** [comment]
Yes. All of the code is in https://github.com/allenai/open-instruct/pull/424 actually, but we will merge it some time soon.
